pages.id/windows/ipconfig.md

```diff
@@ -0,0 +1,24 @@
+# ipconfig
+
+> Menampilkan dan mengatur konfigurasi jaringan dalam sistem operasi Windows.
+> Informasi lebih lanjut: <https://docs.microsoft.com/windows-server/administration/windows-commands/ipconfig>.
+
+- Menunjukkan daftar adaptor jaringan:
+
+`ipconfig`
+
+- Menunjukkan daftar adaptor jaringan secara lengkap:
+
+`ipconfig /all`
+
+- Memperbarui alamat IP sebuah adaptor jaringan:
+
+`ipconfig /renew {{adapter}}`
+
+- Mengosongkan alamat-alamat IP yang disetel dalam sebuah adaptor jaringan:
+
+`ipconfig /release {{adapter}}`
```

Comment:
Please translate the `adapter` work to Indonesian to help non-English speaker.

Reply:
See https://github.com/tldr-pages/tldr/pull/6045/commits/46a932dc57c0f544c126cc1c8cc6b52f60539494

`adaptor` is considered [a standard word](https://kbbi.kemdikbud.go.id/entri/adaptor) of the Indonesian language, so I don't translate `adaptor` further from `adapter`.